### PR TITLE
FLUID-5797: updating the namespace to fluid_2_0_0

### PR DESCRIPTION
**Pull request by jobara**
Now that the version number reflects the next stable release, switching the namespace to match.

https://issues.fluidproject.org/browse/FLUID-5797

### CODE REVIEW

**Comment by cindyli** [comment]
Shall we remove the mentioning of "beta" in ReleaseNotes.md as well?

File: src/components/pager/js/Table.js [hunk 1]
```diff
@@ -436,4 +436,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
         columnDefs: [] // [{key: "columnName", valuebinding: "*.valuePath", sortable: true/false}]
     });
 
-})(jQuery, fluid_2_0_0_beta_1);
\ No newline at end of file
+})(jQuery, fluid_2_0_0);
```
Comment (cindyli) [inline src/components/pager/js/Table.js]: We probably can take this opportunity to add the newline at the end of the file.

File: src/components/uploader/js/HTML5UploaderSupport.js [hunk 2]
```diff
@@ -363,4 +363,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
         }
     });
 
-})(jQuery, fluid_2_0_0_beta_1);
\ No newline at end of file
+})(jQuery, fluid_2_0_0);
```
Comment (cindyli) [inline src/components/uploader/js/HTML5UploaderSupport.js]: Same as above, add the newline at EOF.

File: src/components/uploader/js/MimeTypeExtensions.js [hunk 3]
```diff
@@ -53,4 +53,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
         ppt: "application/vnd.ms-powerpoint",
         pptx: "application/vnd.ms-powerpoint"
     };
-})(fluid_2_0_0_beta_1);
\ No newline at end of file
+})(fluid_2_0_0);
```
Comment (cindyli) [inline src/components/uploader/js/MimeTypeExtensions.js]: Same as above, add the newline at EOF.

File: src/framework/core/js/FluidPromises.js [hunk 4]
```diff
@@ -257,4 +257,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
     };
     
     
-})(jQuery, fluid_2_0_0_beta_1);
\ No newline at end of file
+})(jQuery, fluid_2_0_0);
```
Comment (cindyli) [inline src/framework/core/js/FluidPromises.js]: Same as above, add the newline at EOF.

File: src/framework/core/js/FluidViewDebugging.js [hunk 5]
```diff
@@ -660,4 +660,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
         fluid.debug.browser("body");
     });
     
-})(jQuery, fluid_2_0_0_beta_1);
\ No newline at end of file
+})(jQuery, fluid_2_0_0);
```
Comment (cindyli) [inline src/framework/core/js/FluidViewDebugging.js]: Same as above, add the newline at EOF.

File: src/framework/preferences/css/stylus/FullPrefsEditor.styl [hunk 6]
```diff
@@ -1,4 +1,4 @@
-// UI Option styles for full page layouts both with and without preview
+s// UI Option styles for full page layouts both with and without preview
```
Comment (cindyli) [inline src/framework/preferences/css/stylus/FullPrefsEditor.styl]: typo of the "s"

Comment (jobara) [inline src/framework/preferences/css/stylus/FullPrefsEditor.styl]: Thanks for catching that

**Comment by cindyli** [comment]
Merged @ c9d974c3b2a667f963afd0ba90d55fc4e187bc08

**Pull request merged**